modules/apps/object/object-web/src/main/java/com/liferay/object/web/internal/object/definitions/display/context/util/ObjectCodeEditorUtil.java

```diff
@@ -165,20 +239,26 @@ public static List<HashMap<String, String>> getItems(Locale locale) {
 						"label",
 						LanguageUtil.get(locale, ddmExpressionOperator._key)
 					).put(
-						"tooltip", StringPool.BLANK
+						"tooltip",
```

Comment:
@MariMalta Please, rename all occurences of tooltip with "helpText". I think it will be better to not associate the name of the parameter with a type of element it could be displayed inside.